skin/frontend/base/default/js/TIG/MyParcelBE/checkout_type/onestepcheckout_index_index.js

```diff
@@ -10,7 +10,8 @@ var fnCheckout = {
         var frm = mypajQuery('form');
         clearTimeout(timeout);
         timeout = setTimeout(function () {
-            if (xhr && xhr.readyState != 4) {
```

Comment:
Is er een reden waarom je `xhr.readyState != 4` hebt weggehaald?

Reply:
In [MyParcel NL](https://github.com/myparcelnl/magento1/commit/8eb8bccb7fca78e62c733a7889e541113b169f9d#diff-3e8311cc331334355500ede05944fb19) was er een probleem met de geselecteerde prijs. Dit was de oplossing.

Reply:
Done!